Lauhdutin/@Resources/Frontend/GUI.lua

```diff
@@ -337,6 +387,79 @@ end
 			else
 				return tResult, true
 			end	
+		elseif StartsWith(asPattern, 'random:') then
```

Comment:
- The entire implementation of the `random:` filtering tag includes several unnecessary local variables:
  - `TotalGames` could be replaced with `#tResultR`.
  - The return value of `math.random` does not need to be stored in a variable such as `RandomGame`. It can be used immediately `table.insert(tResult, tResultR[math.random(1, #tResultR)])`. Doing it this way also means that there is no need to iterate through `tResultR`.
- There should be an empty argument variant that returns a random game from the current set of already filtered games (i.e. from `atTable`).